menu/menu.yaml

```diff
@@ -527,22 +527,24 @@
   - Url: nservicebus/rabbitmq
     Title: RabbitMQ Transport
     Articles:
-    - Url: nservicebus/rabbitmq/rabbitmqclient-nuget
-      Title: RabbitMQ.Client NuGet usage
```

Comment:
When deciding where in the menu the new page needed to go, I decided to reorder things as well, because I was unhappy with the current ordering.